src/common/components/radioGroup/radioItem/RadioItem.tsx

```diff
@@ -0,0 +1,30 @@
+import { JSX } from 'react'
+
+import * as Radio from '@radix-ui/react-radio-group'
+import cn from 'classnames'
+
+import { Typography } from '@common/components'
+import s from '@common/components/radioGroup/radioItem/RadioItem.module.scss'
+import { TypographyVariant } from '@common/enums'
+
+export type RadioItem = {
+  value: string
+  labelTitle: string
+  disabled: boolean
+}
+export const RadioElement = ({ value, labelTitle, disabled }: RadioItem): JSX.Element => {
+  return (
+    <div className={s.itemContainer}>
+      <Typography
+        as={'label'}
+        variant={TypographyVariant.Body2}
+        className={cn(s.label, disabled && s.disabled)}
```

Comment:
Attribute className should be the first, higher only attribute ref

Reply:
Done!